tools/createTestReport/create_test_report.py

```diff
@@ -0,0 +1,139 @@
+"""Project specific Markdown converter functions.
+
+    Author: Andreas Merkle (andreas.merkle@newtec.de)
+"""
+
+# pyTRLCConverter - A tool to convert PlantUML diagrams to image files.
+# Copyright (c) 2024 - 2025 NewTec GmbH
+#
+# This file is part of pyTRLCConverter program.
+#
+# The pyTRLCConverter program is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the Free Software Foundation,
+# either version 3 of the License, or (at your option) any later version.
+#
+# The pyTRLCConverter program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with pyTRLCConverter.
+# If not, see <https://www.gnu.org/licenses/>.
+
+# Imports **********************************************************************
+import os
+import shutil
+from pyTRLCConverter.ret import Ret
+from pyTRLCConverter.plantuml import PlantUML
+
+from pyTRLCConverter.markdown_converter import MarkdownConverter
+from pyTRLCConverter.trlc_helper import Record_Object
+
+# Variables ********************************************************************
+
+# Classes **********************************************************************
+
+
+class CustomMarkDownConverter(MarkdownConverter):
```

Comment:
We should use dedicated names also they do not matter functionality wise. But searching for "CustomMarkdownConverter" will lead to multiple definitions which is confusing.

"TestReportMarkdonConverter " ?

Reply:
#34